bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/block/prettyprint/PrettifyJavaScriptReference.java

```diff
@@ -34,7 +35,7 @@ private PrettifyJavaScriptReference() {
 
     @Override
     public List<HeaderItem> getDependencies() {
-        List<HeaderItem> dependencies = Generics2.newArrayList(super.getDependencies());
+        List<HeaderItem> dependencies = super.getDependencies().stream().collect(Collectors.toCollection(ArrayList::new));
```

Comment:
`new ArrayList(super.getDependencies())` ?!

Reply:
thanks :) `Generics2` looks unnecessary to me, but I stuck in couple of places :(